assets/scripts/utils/array.js

```diff
@@ -1,101 +1,124 @@
 
-export function addToArray ( array, value ) {
-    const index = array.indexOf( value );
 
-    if ( index === -1 ) {
-        array.push( value );
+/**
+ * Add item of array if doesn't exist
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to add to the array
+ * @return {void}
+ */
+
+const addToArray = (array, value) => {
+    if (array.indexOf(value) === -1) {
+        array.push(value)
     }
 }
 
-export function arrayContains ( array, value ) {
-    for ( let i = 0, c = array.length; i < c; i++ ) {
-        if ( array[i] == value ) {
-            return true;
-        }
-    }
 
-    return false;
-}
+/**
+ * Check if array contains a value
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to find in the array
+ * @return {boolean}    true if array contains value
+ */
 
-export function arrayContentsMatch ( a, b ) {
-    let i;
+const arrayContains = (array, value) => array.find(item => item === value)
 
-    if ( !isArray( a ) || !isArray( b ) ) {
-        return false;
-    }
 
-    if ( a.length !== b.length ) {
-        return false;
-    }
+/**
+ * Compare two arrays and see if their content match
+ * @param {array}       a   - first array of items
+ * @param {array}       b   - second array of items
+ * @return {boolean}    true if both arrays are the same
+ */
 
-    i = a.length;
-    while ( i-- ) {
-        if ( a[i] !== b[i] ) {
-            return false;
-        }
-    }
+const arrayContentsMatch = (a, b) => a.length === b.length && a.every((v, i) => v === b[i])
 
-    return true;
-}
 
-export function ensureArray ( x ) {
-    if ( typeof x === 'string' ) {
-        return [ x ];
-    }
+/**
+ * Get last item of the given array
+ * @param {array}       array   - array of items
+ * @return {?string}    the last value of the array
+ */
 
-    if ( x === undefined ) {
-        return [];
-    }
+const lastItemOfArray = array => array[array.length - 1]
 
-    return x;
-}
 
-export function lastItem ( array ) {
-    return array[ array.length - 1 ];
-}
+/**
+ * Remove specific value from array
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to remove from the array
+ * @return {void}
+ */
 
-export function removeFromArray ( array, member ) {
-    if ( !array ) {
-        return;
+const removeFromArray = (array, value) => {
+    if (!array) {
+        return
     }
 
-    const index = array.indexOf( member );
-
-    if ( index !== -1 ) {
-        array.splice( index, 1 );
+    const index = array.indexOf(value)
+    if (index !== -1) {
+        array.splice(index, 1)
     }
 }
 
-export function toArray ( arrayLike ) {
-    const array = [];
-    let i = arrayLike.length;
-    while ( i-- ) {
-        array[i] = arrayLike[i];
+
+/**
+ * Convert array like to array
+ * @param {array}   arrayLike   - array like to convert
+ * @return {array}  array converted
+ */
+
+const toArray = arrayLike => {
```

Comment:
This should be replaced with `Array.from()`.

Reply:
If it's to return a function that only does `array => Array.from(array)` we can definitely delete this one